app/controllers/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationController < ActionController::Base
+  skip_before_action :verify_authenticity_token
```

Comment:
- It looks like you forgot to don't skip the authentication for all the controllers, I kindly invite you to add this function to your API service, an authentication should be a requirement to use the endpoint.
- I just try to use the Heroku application without this and I could use it, I kindly invite you to improve this functionality
![image](https://user-images.githubusercontent.com/20563252/186820540-5fa5b47a-15ef-4843-bd3e-56036c86ff78.png)